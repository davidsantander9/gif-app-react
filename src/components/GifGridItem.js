import React from 'react';
import PropTypes from 'prop-types';

const GifGridItem = ({title, url}) => {
    return (
        <div className="card animate__animated animate__fadeIn animate__delay-1s">
            <p>{title}</p>
            <img src={url} alt={title}/>
        </div>
    )
}

GifGridItem.protoTypes = {
    title: PropTypes.string.isRequired,
    url:   PropTypes.string.isRequired,
}

export default GifGridItem;
