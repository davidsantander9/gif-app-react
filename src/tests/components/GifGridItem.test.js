import React from 'react';
import { shallow } from 'enzyme';
import GifGridItem from '../../components/GifGridItem';

describe('<GifGridItem/>  test', () => {

    const title = 'Title';
    const url = 'http//localhost/algo.jpg';

    test('should correctly display the component', () => {
        const wrapper = shallow(<GifGridItem title={ title } url={ url } />);
        expect( wrapper ).toMatchSnapshot();
    });
    


});