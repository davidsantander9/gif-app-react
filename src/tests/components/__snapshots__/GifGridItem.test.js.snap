// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GifGridItem/>  test should correctly display the component 1`] = `
<div
  className="card animate__animated animate__fadeIn animate__delay-1s"
>
  <p>
    Title
  </p>
  <img
    alt="Title"
    src="http//localhost/algo.jpg"
  />
</div>
`;
